### ISSUE TITLE
Chart preview starts at seemingly-random position; only snaps to correct position after some time

### ISSUE THREAD

**Issue by kangalioo** [etternagame/etterna#796]
**Describe the bug**
When activated, the chart preview starts at a some position that does not match the music playing. Only after some time (~10s+), it suddenly snaps to the correct position and plays in sync with the music.

**To Reproduce**
1. Activate chart preview
2. Look at chart preview

**Expected behavior**
The chart preview should start at the correct position immediately.

**Desktop (please complete the following information):**
 - OS: Arch Linux 64bit
 - Version: 0.69.1 from git

**Additional context**
I feel like this issue does not happen on Windows. From what I've seen, chart preview works fine on other people's computers.

Also, this issue is not new to 0.69. Actually, I can't even remember when I encountered it the first time, or if there was even any version at all in which this bug didn't occur. If needed, I can try out some of the older version in an attempt to pinpoint the bug source.

**Comment by poco0317** [comment]
Chart preview position is dependent on Gamestate's song position which updates according to the last playing sound. If a sound is in progress of loading or something breaks, this is instead attached to an update loop which can be sped up with Tab. That will explain why sometimes you can hold Tab and the preview moves fast.

**Comment by poco0317** [comment]
Now that I'm actually reading this, how does it actually take 10 seconds to snap back to being synced? It should sync as soon as the music starts playing (pretty much the point you hear it).

**Comment by poco0317** [comment]
I've removed most jarring music resetting so this issue may have disappeared. Check the latest builds (ec4829238772aed67b6bef73058cdd664a846838 or newer)

**Comment by kangalioo** [comment]
Still happens on every single chart. Although it doesn't seem to take 10s+ anymore for the arrows to jump to their correct position, just 1-2 seconds now - maybe I just got lucky.

**Comment by nico-abram** [comment]
@kangalioo Do you know if this is still an issue?

**Comment by kangalioo** [comment]
Yes, it still happens in 0.71.2 08773a50b4. Mostly it takes 1-2s to snap back, but sometimes 10s or more

**Comment by Beethoven-n** [comment]
still happens on v0.72.3

**Comment by sliceofpizzaxd** [comment]
still an issue on v0.74.4
